test/recipes/30-test_evp_data/evpmac_common.txt

```diff
@@ -239,6 +239,31 @@ Input = "Test that SHAKE128 fails"
 Key = 000102030405060708090a0b0c0d0e0f101112131415161718191a1b1c1d1e1f
 Result = MAC_INIT_ERROR
 
+Title = HMAC-SM3 from GM/T 0042-2015 Appendix D.3
+
+MAC = HMAC
```

Comment:
Add ````Availablein = default```` above every MAC = HMAC line..
This makes the tests only run for the default provider. SM3 is not a FIPS algorithm

Reply:
OR create a separate file evpmac_sm3.txt and add it into 30-test_evp.t at line 108 (which is a list of tests that just run with the default provider)